.gitignore

```diff
@@ -15,6 +15,7 @@ lib/
 pylib/src/
 pylib/cqlshlib/serverversion.py
 !lib/cassandra-driver-internal-only-*.zip
+!lib/puresasl-*.zip
```

Comment:
Do you need to add here `datastax_db_*` as well?

Reply:
Not needed in this project (this library will only be included in HCD).

Reply:
Aha right. In HCD it would make sense but not here.